site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -1,27 +1,39 @@
 import { useMachine } from "@xstate/react"
-import { TemplateVersionParameter } from "api/typesGenerated"
```

Comment:
I'm not sure why but in my case "Open in Code" component is pretty big. Larger than the parameter form on the left. Is it intended?

<img width="1369" alt="Screenshot 2023-07-26 at 10 47 21" src="https://github.com/coder/coder/assets/14044910/679d2b97-2bb5-428d-bfb4-eb244e81270b">

Reply:
Yes, it is. But I think I can improve that 🤔